packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -65,6 +66,26 @@ Please make sure you are running "shorebird init" from within your Flutter proje
       return ExitCode.software.code;
     }
 
+    final orgs = await codePushClientWrapper.getOrganizationMemberships();
+    if (orgs.isEmpty) {
+      logger.err('''
+You do not have any organizations.
+Please create an organization on the Shorebird website before continuing.
+''');
+      return ExitCode.software.code;
+    }
```

Comment:
Is this even possible?  Or does the "personal" organization not show up in the "organization memberships" list?

Reply:
It shouldn't be, but I wanted to be sure we give people clear instructions in case it somehow happens. 

This is a draft, btw, so I wouldn't spend too much time on it. Will let you know when it's ready.